package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1729,7 +1729,99 @@ def radius_of_gyration(group, wrap=False, **kwargs):
     @warn_if_not_unique
     @_pbc_to_wrap
     @check_atomgroup_not_empty
-    def shape_parameter(group, wrap=False):
+    def gyration_moments(group, wrap=False, unwrap=None, compound='group'):
+        r"""Moments of the gyration tensor.
+
+        The moments are defined as the eigenvalues of the gyration
+        tensor.
+
+        .. math::
+        
+            \mathsf{T} = \frac{1}{N} \sum_{i=1}^{N} (\mathsf{r_{i}} - 
+                \mathsf{r_{COM}})(\mathsf{r_{i}} - \mathsf{r_{COM}})
+
+        Where :math:`r_{COM}` is the center of mass.
```

Comment:
Typeset consistently, please
* vectors with `\mathbf{r}`, tensors with `\mathsf{T}`.
* subscript labels in roman `\mathbf{r}_\mathrm{COM}` (but indices just as normal math quantities0